axis_backend/apps/clients/serializers/industry_serializer.py

```
@@ -0,0 +1,241 @@
+
+        Args:
+            value: Industry name
+
+        Returns:
+            Validated and stripped name
+
+        Raises:
+            serializers.ValidationError: If name is empty
+        """
+        if not value or not value.strip():
+            raise serializers.ValidationError("Industry name cannot be empty")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)